walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,121 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void optionalAttendeeIsNotConsidered() {
```

Comment:
nit: rename this test to indicate why the optional attendee is not considered:
optionalAttendeeWithConflict_returnMandatoryOnly

Reply:
Okay!